### PR TITLE
HPCC-14290 Add option for thor slave in ZAP

### PR DESCRIPTION
**Pull request by miguelvazq**
Add ability to pull down the thor slave logs along with the ZAP report.

Signed-off-by: Miguel Vazquez miguel.vazquez@lexisnexis.com

### CODE REVIEW

**Comment by miguelvazq** [comment]
@GordonSmith Please review.

**Comment by GordonSmith** [comment]
Looks good.

File: esp/src/eclwatch/WUDetailsWidget.js [hunk 1]
```diff
@@ -404,6 +406,7 @@ define([
                         context.thorProcess.set("value", targetData[0].ClusterGroup);
                 } else {
                    context.slaveLogs.set("disabled", true);
+                   context.includeSlaveLogsCheckbox.set("disabled", true);
```
Comment (richardkchapman) [inline esp/src/eclwatch/WUDetailsWidget.js]: What's the differernce between slaveLogs and includeSlaveLogsCheckbox ?

Comment (GordonSmith) [inline esp/src/eclwatch/WUDetailsWidget.js]: The checkbox is to include the slave logs in the ZAP report - the other is complete drop down menu option.

Comment (GordonSmith) [inline esp/src/eclwatch/WUDetailsWidget.js]: IOW you can download the slave logs independently of a ZAP report (which may be overkill?):  
image

**Comment by miguelvazq** [comment]
@g-pan Does this need documentation?

**Pull request merged**